### PR TITLE
docs(analytics-js-service-worker): document usage for serverless runtime

### PR DESCRIPTION
**Pull request by bardisg**
## PR Description

Please include a summary of the change along with the relevant motivation and context.

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Comment by bardisg** [comment]
Resolves SDK-214

File: examples/chrome-extension/USAGE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Chrome Extension Usage
 
 RudderStack JS SDK can be used in Chrome Extensions with manifest v3, both as a content script or as a background script
-service worker.
+service worker. It exposes the same interface and features as the [nodeJS SDK](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-node-sdk/)
```
Comment (saikumarrs) [inline examples/chrome-extension/USAGE.md:4]:
```suggestion
service worker. It exposes the same interface and features as the [NodeJS SDK](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-node-sdk/)
```

File: examples/serverless/USAGE.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Serverless Usage
+
+RudderStack JS SDK service worker can be used in serverless runtimes like Cloudflare Workers or Vercel Edge functions.
+It exposes the same interface and features as the [nodeJS SDK](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-node-sdk/)
```
Comment (saikumarrs) [inline examples/serverless/USAGE.md:4]:
```suggestion
It exposes the same interface and features as the [NodeJS SDK](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-node-sdk/)
```

File: packages/analytics-js-plugins/.size-limit.js [hunk 3]
```diff
@@ -13,6 +13,6 @@ module.exports = [
     name: 'Remote Module Federated Plugins - CDN',
     path: 'dist/cdn/modern/plugins/rsa-plugins-*.js',
     gzip: true,
-    limit: '15 KiB',
+    limit: '19 KiB',
```
Comment (saikumarrs) [inline packages/analytics-js-plugins/.size-limit.js:16]: Why did we bump it? 15 KiB seems a reasonable limit.

**Review by saikumarrs — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**